src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -1133,14 +1128,7 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
           size='lg'
           colorScheme={errorCopy ? 'red' : 'blue'}
           onClick={handleSubmit}
-          isDisabled={
-            !confirmedQuote ||
-            (isEstimatedPoolAssetFeesDataError && opportunityType === AsymSide.Asset) ||
-            (isEstimatedRuneFeesDataError && opportunityType !== AsymSide.Asset) ||
-            !validInputAmount ||
-            isSweepNeededLoading ||
-            Boolean(errorCopy)
-          }
```

Comment:
monkey patch reverted in  6668e4481d0a8e71dbad7bb8dac69cbaca5f77aa - revert the revert of it if you want to test with withdraws lower than minimum